tests/unittests/conftest.py

```diff
@@ -63,6 +63,19 @@ def fake_filesystem(mocker, tmpdir):
     yield str(tmpdir)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def disable_sysfs_net(request, tmpdir_factory):
+    if hasattr(request, "param") and getattr(request, "param") is False:
```

Comment:
Nit: The `hasattr` call isn't needed, right? `getattr` will return `None` if the argument doesn't exist?

Reply:
True story.